download.md

```diff
@@ -337,9 +319,6 @@ These modules may require accounts with a service provider.<br/>
 </div></div>{% endfor %}
 </div>
 
-{% for page in module_pages %}{% if page.enable_download == false%}{% continue %}{% endif %}{% if page.min_js_version %}<input type="hidden" class="version-restriction" moduleCode="{{ page.module_code }}" name="min-version" value="{{ page.min_js_version }}">{% endif %}
```

Comment:
What was this used for?

Reply:
This was used to notify the user about the module not being included because the min version was not met.
I moved it to the existing module inputs instead:
https://github.com/prebid/prebid.github.io/pull/4963/files#diff-c8eec1db450201507b0137188c7c58afd76e678c22c2de1e8b4a59e9a6a9bb41R312

Reply:
Smooth :+1: